File: examples/python/object_removal.py
import asyncio
import os
from ailabtools import AILabClient


async def main():
    client = AILabClient(api_key=os.environ["AILAB_API_KEY"])
    with open("./photo.jpg", "rb") as image, open("./mask.png", "rb") as mask:
        result = await client.image.remove_objects({"image": image, "mask": mask})
    print(result["data"]["image_url"])
    await client.aclose()


asyncio.run(main())

File: examples/node/object-removal.ts
import { readFileSync } from "node:fs";
import { AILabClient } from "ailabtools";

const client = new AILabClient({ apiKey: process.env.AILAB_API_KEY! });

async function main() {
  const result = await client.image.removeObjects({
    image: readFileSync("./photo.jpg"),
  });

  console.log(result.data?.image_url);
}

main().catch((error) => {
  console.error(error);
  process.exit(1);
});
